versions.json

```diff
@@ -1,9 +1,11 @@
 {
-  "kotlinVersion": "1.9.10",
-  "agpVersion": "7.3.0",
-  "composeVersion": "1.5.10",
-  "androidxAppCompat": "1.6.1",
-  "androidxCore": "1.10.1",
+  "kotlinVersion": "2.0.20",
+  "agpVersion": "8.5.0",
+  "composeVersion": "1.6.11",
+  "androidxAppCompat": "1.7.0",
+  "androidxCore": "1.13.1",
   "ktor": "2.3.6",
-  "koin": "3.4.0"
+  "koin": "3.4.0",
+  "ktorfit": "1.12.0",
+  "napier": "2.6.1"
```

Comment:
What's napier again?

Reply:
I also see this being used anywhere.

Reply:
sorry

I sent the wrong PR

I mainly want to change the versions of various libs (especially the kotlin part, which I want to upgrade to 2.0.20 or above)

I also added my commonly used repos and libs, and then sent the PR without distinguishing them.

Mainly I want to know if this plugin can be modified to support kotlin 2.0 or above and more lib and mvvm templates?